project/dexter-monitor/routes/user.js

```diff
@@ -204,4 +204,144 @@ exports.getUserCountByProjectName = function(req, res) {
             log.error(err);
             res.send({status:"fail", errorMessage: err.message});
         });
+};
+
+function loadUserStatusList() {
+    const sql = "SELECT * FROM DexterUserList";
+    return database.exec(sql)
+        .then((rows) => {
+            return rows;
+        })
+        .catch((err) => {
+            log.error(err);
+            return [];
+        });
+}
+
+function loadUserList() {
+    return project.getDatabaseNameList()
+        .then((rows) => {
+            const dbNameList = _.map(rows, 'dbName');
+            let allRows = [];
+            let promises = [];
+
+            dbNameList.forEach((dbName) => {
+                promises.push(new Promise((resolve, reject) => {
+                    getUserListByProjectDatabaseName(dbName)
+                        .then((rows) => {
+                            allRows = _.union(allRows, rows);
+                            resolve();
+                        })
+                        .catch((err) => {
+                            log.error(err);
+                            reject();
+                        })
+                }));
+            });
+
+            return Promise.all(promises)
+                .then(() => {
+                    allRows = _.uniq(allRows, (row) => {
+                        return row.userId;
+                    });
+                    return allRows;
+                })
+                .catch((err) => {
+                    log.error(err);
+                    return [];
+                });
+        })
+        .catch((err) => {
+            log.error('Failed to get the DB name list: ' + err);
+            return [];
+        });
+}
+
+function updateUserStatusList() {
+    let promises = [];
+    let userStatusList = [];
+    let userList = [];
+
+    promises.push(new Promise((resolve, reject) => {
+        loadUserStatusList()
+            .then((rows) => {
+                userStatusList = rows;
+                resolve();
+            })
+            .catch((err) => {
+                log.error(err);
+                reject();
+            })
+    }));
+    promises.push(new Promise((resolve, reject) => {
+        loadUserList()
+            .then((rows) => {
+                userList = rows;
+                resolve();
+            })
+            .catch((err) => {
+                log.error(err);
+                reject();
+            })
+    }));
+
+    return Promise.all(promises)
+        .then(() => {
+            return Promise.map(userStatusList, (row) => {
+                if (_.findIndex(userList, (user) => user.userId == row.userId) >= 0) {
+                    row.dexterYn = 'Y';
```

Comment:
`dexterYn` has three values now, `Y` for installed developer, `N` for not installed developer and `U` for inapplicable developer. I think that both the `wasInstalled` and `hasDexterClient` cannot cover the meaning of value `U`. We(@Min-Ho-Kim and @Minjung-Baek) should think about it more.